scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -397,9 +394,11 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
 
         vd = vd.drop('allele_NO', 'allele_YES', 'allele_is_snp', 'allele_OK')
 
-        if unphase:
-            vd = vd.annotate_entries(LGT=vd.LGT.unphase())
-        vd = vd.select_entries(GT=hl.vds.lgt_to_gt(vd.LGT, vd.LA), **vd.entry)
+        vd = vd.select_entries(
+            GT=hl.vds.lgt_to_gt(vd.LGT, vd.LA),
+            PGT=hl.vds.lgt_to_gt(vd.LPGT, vd.LA),
+            **vd.entry
+        )
```

Comment:
i think we can remove this whole section. If we have LGT and LPGT, having GT and PGT are overkill esp if Lee isn't asking for them explicitly as he was in Delta